src/uucore/src/lib/mods/backup_control.rs

```diff
@@ -0,0 +1,96 @@
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+pub static BACKUP_CONTROL_VALUES: &[&str] = &[
+    "simple", "never", "numbered", "t", "existing", "nil", "none", "off",
+];
+
+pub static BACKUP_CONTROL_LONG_HELP: &str = "The backup suffix is '~', unless set with --suffix or SIMPLE_BACKUP_SUFFIX. Here are the version control values:
+
+none, off
+    never make backups (even if --backup is given)
+
+numbered, t
+    make numbered backups
+
+existing, nil
+    numbered if numbered backups exist, simple otherwise
+
+simple, never
+    always make simple backups";
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+pub enum BackupMode {
+    NoBackup,
+    SimpleBackup,
+    NumberedBackup,
+    ExistingBackup,
+}
+
+pub fn determine_backup_suffix(supplied_suffix: Option<&str>) -> String {
```

Comment:
`clap` can also handle environment variables and default values: https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.env

Although I can also understand if you implemented it here to ensure that the behaviour between `cp` and `mv` is the same.

Reply:
Totally. I would rather have this functionality in `backup_control.rs` as it's easy to miss in other utilities implementing backups. Let me know if you require this change and I'll happily make it.